java/arcs/core/data/testdata/WriterReaderExample.kt

```diff
@@ -0,0 +1,30 @@
+/* ktlint-disable */
+@file:Suppress("PackageName", "TopLevelName")
+
+package arcs.core.data.testdata
+
+//
+// GENERATED CODE -- DO NOT EDIT
+//
+
+import arcs.core.data.*
+import arcs.core.storage.*
+
+object WritingRecipePlan : Plan(
```

Comment:
The point would be to generate the result for WriterReaderExample.arcs in the test and validate it is equal to this file. Then as we change the recipe2plan we update this golden file. This allows us to visualize much easier what is the impact of changes in the code review as well as for folks to see how does the output of this tool looks like without even running it, which is nice.